atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -2008,10 +2008,21 @@ private void putMetadataAndMaybeAlterTables(
                 return null;
             });
         } catch (Exception e) {
-            throw Throwables.unwrapAndThrowAtlasDbDependencyException(e);
+            log.error("Failed to apply the table metadata changes for the following Cfdefs {}",
+                    SafeArg.of("cfdef", getAllCfdefsString(updatedCfs)), e);
+
+//            throw Throwables.unwrapAndThrowAtlasDbDependencyException(e);
         }
     }
 
+    private String getAllCfdefsString(Collection<CfDef> updatedCfs) {
+        StringBuilder sb = new StringBuilder("CfDefs(");
```

Comment:
you're going to get "... Cfdefs CfDefs(...)" which may a look a bit odd - can updated after the major issue is resolved though.